### PR TITLE
PLT-7810 Track if an APIv3 endpoint was used in the last day

### PR DESCRIPTION
**Pull request by jwilander**
#### Summary
Track if an APIv3 endpoint was used in the last day. /cc @jasonblais 

#### Ticket Link
https://mattermost.atlassian.net/browse/PLT-7810

### CODE REVIEW

File: model/client.go [hunk 1]
```diff
@@ -18,6 +18,8 @@ import (
 	l4g "github.com/alecthomas/log4go"
 )
 
+var UsedApiV3 bool
```
Comment (coreyhulen) [inline model/client.go]: Don't think this is thread safe.  You might want to use an atomic int instead.

Comment (jwilander) [reply model/client.go]: Didn't think it would really matter but I'll make it atomic anyways

**Comment by jwilander** [comment]
@coreyhulen updated

**Comment by jwilander** [comment]
Not super familiar with "sync/atomic" so if it's better to read to check if it's set instead of writing every time let me know

**Pull request merged**